ideas/276-fast-discovery.md

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
Nitpick: when I click "View" in GH to see this file formatted, this YAML part displays as raw. I guess it should display as a table like [here](https://github.com/status-im/ideas/blob/master/ideas/145-identity.md).

Reply:
yeah i thought it will be formatted like that. cant figure out why

Reply:
fixed